stack-9.0.2.yaml

```diff
@@ -20,7 +20,7 @@ packages:
 - ./plugins/hls-retrie-plugin
 - ./plugins/hls-splice-plugin
 # - ./plugins/hls-tactics-plugin
```

Comment:
i guess we should enable tactics here too!

Reply:
oh good catch